src/test/denialofservice_tests.cpp

```diff
@@ -53,7 +53,9 @@ BOOST_AUTO_TEST_CASE(outbound_slow_chain_eviction)
     // Disable inactivity checks for this test to avoid interference
     static_cast<ConnmanTestMsg*>(connman.get())->SetPeerConnectTimeout(99999s);
     auto peerLogic = PeerManager::make(chainparams, *connman, *m_node.addrman, nullptr,
-                                       *m_node.chainman, *m_node.mempool, false);
+                                       *m_node.chainman, *m_node.mempool,
+                                       /*std::unique_ptr<TxReconciliationTracker> txreconciliation=*/nullptr,
```

Comment:
In "p2p: Announce reconciliation support"

Nit: listing the type as well seems a bit excessive to me, and not something we do elsewhere either (here and further in the commit).